sympy/integrals/tests/test_transforms.py

```diff
@@ -505,6 +505,17 @@ def test_laplace_transform():
         ((s - 1)/((s - 1)**2 + 1), -oo),
     ]
 
+    # Delta function
+    assert LT(DiracDelta(t),t,s) == (1, -oo, True)
+    assert LT(DiracDelta(a*t),t,s) == (1/a, -oo, True)
```

Comment:
In this test `a` is declared with `positive=True`. What happens if `a` is just declared as e.g. `Symbol('a', real=True)`?

Reply:
This also gives `(1/a, -oo, True)`.